hooks/post-command

```diff
@@ -47,6 +176,9 @@ main() {
   local ci_env
   ci_env=$(bash <(curl -s -S --connect-timeout 10 --retry 3 --retry-delay 10 https://codecov.io/env))
 
+  codecov_command=./codecov
+  get_codecov_uploader
```

Comment:
This means that on each run a new uploader is downloaded; always `latest` currently. I wonder if we can allow a fixed version set by the plugin config and then cache away that particular version. Similar to what is done here: https://github.com/joscha/sauce-connect-buildkite-plugin/blob/master/hooks/pre-command#L30 

That way the uploader stays stable longer and does not rely on the network fetch each time nor will cause trouble (read massive amounts of requests) when there are many many builds run from one subnet.

Reply:
Fixed in https://github.com/joscha/codecov-buildkite-plugin/commit/388c47dc57d487f625544c2c9532c6b66cc2b2e2
I have added two new configs: `uploader_version` and `tmp_dir` with proper defaults.
The downloaded binary will be cached accordingly.